### PR TITLE
NM|ME: Download link to redirect to s3

### PR DESCRIPTION
**Pull request by urbanslug**
A better way to set the download link while viewing a photo at full size.

### CODE REVIEW

File: src/hatti/views/photos.cljs [hunk 1]
```diff
@@ -91,6 +78,11 @@
                                 js/PhotoSwipeUI_Default
                                 (clj->js photos)
                                 options)]
+    (set! (.. gallery
+              -options
+              -getImageURLForShare)
+          (fn [share-button-data]
+            (or (aget gallery "currItem" "original-src") "")))
```
Comment (royrutto) [inline src/hatti/views/photos.cljs]: what happens when its set to null instead of a blank string?

Comment (ivermac) [inline src/hatti/views/photos.cljs]: In their [docs](http://photoswipe.com/documentation/options.html), the `getImageURLForShare` function by default returns the `src` attribute of the currently selected item or an empty string. We decided to go with the same approach of returning either the `original-src` or an empty string. We are not sure what would happen if we returned null

Comment (ivermac) [inline src/hatti/views/photos.cljs]: Also, `src` attribute will usually have an empty string or a string value but not a null value. May be that's why the function returns a string value

Comment (royrutto) [inline src/hatti/views/photos.cljs]: cool

**Pull request merged**